docs/UserGuide.md

```diff
@@ -112,15 +114,20 @@ number) into contact book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS nk/NEXT_KIN nkp/NEXT_KIN_PHONE [fp/FINANCIAL_PLAN] [t/TAG]…​`
 
+* Adding a person with the exact same name (case-sensitive) as a person currently in the contact book will cause the
+    command to fail.
+* To prevent accidentally adding duplicates, you can use [Find](#locating-persons-by-name-financial-plan-andor-tag--find)
+    to check if you have already added the person already.
+
 Acceptable Values:
-1. NAME - any value is possible
-2. PHONE_NUMBER - Numbers (0-9) and symbols, but no alphabets
+1. NAME - Alphanumeric or space characters only
+2. PHONE_NUMBER - Numbers (0-9) only and at least 3 digits long
 3. EMAIL - string in valid email format
 4. ADDRESS - any value is possible
-5. NEXT_KIN - any value is possible
-6. NEXT_KIN_PHONE - Numbers (0-9), and symbols, no alphabets
-7. FINANCIAL_PLAN - Alphanumeric or Space characters
-8. TAG - Alphanumeric
+5. NEXT_KIN - Alphanumeric or space characters only
+6. NEXT_KIN_PHONE - Numbers (0-9) only and at least 3 digits long
+7. FINANCIAL_PLAN - Alphanumeric or space characters only
+8. TAG - Alphanumeric characters only
```

Comment:
Maybe can put (whtie space characters not allowed) to make it clearer since user may not understand that Alphanumeric chars does not include white space.

Reply:
Can add to the glossary I guess